apps/web/src/components/ClickableLogo/ClickableLogo.stories.tsx

```diff
@@ -0,0 +1,8 @@
+import ClickableLogo from './ClickableLogo';
+
+export default {
+  title: 'Components/ClickableLogo',
+  component: ClickableLogo,
+};
+
+export const Default = () => <ClickableLogo />;
```

Comment:
Since we aren't using composition for this component can you use the `template.bind({})` syntax?